### PR TITLE
Minor medbot text changes.

### PR DESCRIPTION
**Pull request by Winterous**
Corrected a grammatical error in finding new knowledge: Used to say, for example, "efficiency increased by 200%" when it was doubling the heal, which is incorrect. Also changed the phrasing a little to fit and not sound wonky.

There were two adjacent synonyms in the same sentence when a tipped bot rights itself, I split the difference between them.

Changed an outdated prompt to player-controlled medbots trying to heal people that aren't hurt enough.

## Changelog
:cl:
fix: Local master coder makes minor text edits to Medbots, deftly correcting the misleading "new knowledge" dialogue, as well as two other changes. Truly an incredible feat of coding excellence.
/:cl:

### CODE REVIEW

**Comment by Winterous** [comment]
God I hope I haven't fucked something up here, been two years since I've actually used github for more than browsing, interface is a little tricky to work with.

**Comment by LemonInTheDark** [comment]
You should add a cl so the playerbase finds out about this. Otherwise, +1

**Comment by Winterous** [comment]
Wait the changelogs are done in the pr? I assumed that someone added it when they merged.

**Comment by LemonInTheDark** [comment]
Yep, lemme grab ya the block

**Comment by LemonInTheDark** [comment]
## About The Pull Request



## Why It's Good For The Game



## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:
use this template

File: code/modules/mob/living/simple_animal/bot/medbot.dm [hunk 1]
```diff
@@ -203,7 +203,7 @@
 		if(tech_boosters)
 			heal_amount = (round(tech_boosters/2,0.1)*initial(heal_amount))+initial(heal_amount) //every 2 tend wounds tech gives you an extra 100% healing, adjusting for unique branches (combo is bonus)
 			if(oldheal_amount < heal_amount)
-				speak("Surgical Knowledge Found! Efficiency is increased by [round(heal_amount/oldheal_amount*100)]%!")
+				speak("New Knowledge Found! Surgical efficiency increased to [round(heal_amount/oldheal_amount*100)]%!")
```
Comment (ATH1909) [inline code/modules/mob/living/simple_animal/bot/medbot.dm:206]:
```suggestion
				speak("Surgical knowledge found! Efficiency multiplied by [round(heal_amount/oldheal_amount*100)]%!")

```

File: code/modules/mob/living/simple_animal/bot/medbot.dm [hunk 2]
```diff
@@ -539,7 +539,7 @@
 
 		if(!treatment_method && emagged != 2) //If they don't need any of that they're probably cured!
 			if(C.maxHealth - C.get_organic_health() < heal_threshold)
-				to_chat(src, "<span class='notice'>[C] is healthy! Your programming prevents you from injecting anyone without at least [heal_threshold] damage of any one type ([heal_threshold + 5] for oxygen damage.)</span>")
+				to_chat(src, "<span class='notice'>[C] is healthy! Your programming prevents you from tending anyone without at least [heal_threshold] damage of any one type ([heal_threshold + 5] for oxygen damage.)</span>")
```
Comment (ATH1909) [inline code/modules/mob/living/simple_animal/bot/medbot.dm:542]:
```suggestion
				to_chat(src, "<span class='notice'>[C] is healthy! Your programming prevents you from tending the wounds of anyone without at least [heal_threshold] damage of any one type ([heal_threshold + 5] for oxygen damage.)</span>")

```

**Comment by Winterous** [comment]
Changed the wording of the research upgrade a bit, and made it show the multiplier of the original healing, rather than the present healing. Makes more sense to state how much they've improved total instead of how much they've improved just now.

Fuck, should I show both present improvement and overall improvement?

File: code/modules/mob/living/simple_animal/bot/medbot.dm [hunk 3]
```diff
@@ -203,7 +203,7 @@
 		if(tech_boosters)
 			heal_amount = (round(tech_boosters/2,0.1)*initial(heal_amount))+initial(heal_amount) //every 2 tend wounds tech gives you an extra 100% healing, adjusting for unique branches (combo is bonus)
 			if(oldheal_amount < heal_amount)
-				speak("Surgical Knowledge Found! Efficiency is increased by [round(heal_amount/oldheal_amount*100)]%!")
+				speak("New Knowledge Found! Surgical efficacy improved to [round(heal_amount/initial(heal_amount)*100)]%!")
```
Comment (ATH1909) [inline code/modules/mob/living/simple_animal/bot/medbot.dm:206]:
```suggestion
				speak("New knowledge found! Surgical efficacy improved to [round(heal_amount/initial(heal_amount)*100)]%!")

```

**Comment by Winterous** [comment]
Ok in the last update I opened it up in VScode and changed the line ending style from CRLF to LF, which is what other files seem to be using. But it still says there's a conflict?

**Comment by Winterous** [comment]
Closing because making a new one with a different branch because these conflict issues make no sense to me.

**Pull request closed without merging**